packages/@o3r/design/src/core/design-token/design-token-specification.interface.ts

```diff
@@ -134,8 +134,7 @@ export interface DesignTokenTypeStrokeStyle<T extends DesignTokenTypeStrokeStyle
 type DesignTokenTypeBorderValue = {
   color: string;
   width: string;
-  // eslint-disable-next-line @typescript-eslint/no-redundant-type-constituents
-  style: string | DesignTokenTypeStrokeStyleValue;
+  style: DesignTokenTypeStrokeStyleValue;
```

Comment:
```suggestion
  style: Exclude<DesignTokenTypeStrokeStyleValue, string> | string;
```

Reply:
We discussed and `string |` was actually a mistake

Reply:
maybe should be done in an another PR then 
else we may think at some point that we lost this during eslint fixes

Reply:
PR done on #2362